src/shared/data/databaseInsertActions.js

```diff
@@ -34,7 +34,7 @@ export const upsertTweeter = (db, tweeter) => {
 export const upsertTweet = (db, tweet) => (
   runQueryOnImmutableObject(
     db,
-    'UPDATE tweet SET id=:id, content=:content, date=:date, likes=:likes, retweets=:retweets, longitude=:longitude, latitude=:latitude UPSERT WHERE id=:id',
+    'UPDATE tweet SET id=:id, content=:content, date=:date, likes=:likes, retweets=:retweets, longitude=:longitude, latitude=:latitude, contains_a_quoted_tweet=:contains_a_quoted_tweet UPSERT WHERE id=:id',
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/winwardo/solid-octo-disco/file/2364574296/issues/source?bid=3148782&fileBranchId=3220697#l37)